book/src/developers/contributing/releases/deployment.md

```diff
@@ -53,21 +53,25 @@ This step directs Ansible to use the current master version of trin. Read [about
 - Activate the virtual environment in the cluster repo: `. venv/bin/activate`
 - Make sure you've pulled the latest master branch of the deployment scripts, to include any recent changes: `git pull origin master`
 - Go into the Portal section of Ansible: `cd portal-network/trin/ansible/`
-- Run the deployment: `ansible-playbook playbook.yml --tags trin`
+- Run the deployment:
+    - Trin nodes:
+        - `ansible-playbook playbook.yml --tags trin`
+    - Testnet nodes (check with the team if there is a reason not to update them):
+        - `ansible-playbook playbook.yml --tags testnet`
```

Comment:
```suggestion
    - State network nodes (check with the team if there is a reason not to update them):
        - `ansible-playbook playbook.yml --tags state-network`
```

I forgot to rename the tag in my last cluster PR we discussed renaming this tag to be state related and not testnet related since this nodes are no longer used for the testnet